Clinical-Genomics/scout
# update PyYaml requirement to use version 5.1

The following error was occurring when running scout cli commands from hasta:

```
2019-05-16 10:14:01 hasta.scilifelab.se scout.commands.base[149323] INFO Running scout version 4.4.1
Traceback (most recent call last):
  File "/home/proj/bin/conda/envs/S_main/bin/scout", line 11, in <module>
    sys.exit(cli())
  File "/home/proj/bin/conda/envs/S_main/lib/python3.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/proj/bin/conda/envs/S_main/lib/python3.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/proj/bin/conda/envs/S_main/lib/python3.7/site-packages/click/core.py", line 1063, in invoke
    Command.invoke(self, ctx)
  File "/home/proj/bin/conda/envs/S_main/lib/python3.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/proj/bin/conda/envs/S_main/lib/python3.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/proj/bin/conda/envs/S_main/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/proj/bin/conda/envs/S_main/lib/python3.7/site-packages/scout/commands/base.py", line 69, in cli
    cli_config = yaml.load(in_handle, Loader=yaml.FullLoader)
AttributeError: module 'yaml' has no attribute 'FullLoader'
```
This PR fixes there error by forcing the installation of a newer version of PyYaml

**How to test:**
install the branch on hasta cli stage as user hiseq.clinical:
``` 
bash update-scout-cli-stage.sh fix_PyYaml_requirement
```
start stage environment with 'us' command. And finally run any cli command, for instance:
``` 
scout view diseases
``` 

You shouldn't see any error.

- [x] code reviewed by @patrikgrenfeldt 
- [x] tested on hasta stage by @ingkebil 
- [x] merge approved by @dnil
